2023-07-26 21:46:48.215 UTC [1177621] LOG:  starting PostgreSQL 15.3 on x86_64-pc-linux-gnu, compiled by gcc (Ubuntu 11.3.0-1ubuntu1~22.04.1) 11.3.0, 64-bit
2023-07-26 21:46:48.215 UTC [1177621] LOG:  listening on IPv4 address "127.0.0.1", port 5432
2023-07-26 21:46:48.221 UTC [1177621] LOG:  listening on Unix socket "/tmp/.s.PGSQL.5432"
2023-07-26 21:46:48.230 UTC [1177624] LOG:  database system was shut down at 2023-07-26 21:46:47 UTC
2023-07-26 21:46:48.236 UTC [1177621] LOG:  database system is ready to accept connections
2023-07-26 21:46:48.405 UTC [1177640] ERROR:  cannot transform non-hash Perl value to hstore
2023-07-26 21:46:48.405 UTC [1177640] CONTEXT:  PL/Perl function "test2"
2023-07-26 21:46:48.405 UTC [1177640] STATEMENT:  SELECT test2();
2023-07-26 21:46:48.407 UTC [1177640] ERROR:  cannot transform non-hash Perl value to hstore
2023-07-26 21:46:48.407 UTC [1177640] CONTEXT:  PL/Perl function "test2"
2023-07-26 21:46:48.407 UTC [1177640] STATEMENT:  SELECT test2();
2023-07-26 21:46:48.607 UTC [1177648] ERROR:  type "foo" does not exist
2023-07-26 21:46:48.607 UTC [1177648] STATEMENT:  CREATE TRANSFORM FOR foo LANGUAGE plperl (FROM SQL WITH FUNCTION hstore_to_plperl(internal), TO SQL WITH FUNCTION plperl_to_hstore(internal));
2023-07-26 21:46:48.608 UTC [1177648] ERROR:  language "foo" does not exist
2023-07-26 21:46:48.608 UTC [1177648] STATEMENT:  CREATE TRANSFORM FOR hstore LANGUAGE foo (FROM SQL WITH FUNCTION hstore_to_plperl(internal), TO SQL WITH FUNCTION plperl_to_hstore(internal));
2023-07-26 21:46:48.608 UTC [1177648] ERROR:  return data type of FROM SQL function must be internal
2023-07-26 21:46:48.608 UTC [1177648] STATEMENT:  CREATE TRANSFORM FOR hstore LANGUAGE plperl (FROM SQL WITH FUNCTION hstore_out(hstore), TO SQL WITH FUNCTION plperl_to_hstore(internal));
2023-07-26 21:46:48.608 UTC [1177648] ERROR:  first argument of transform function must be type internal
2023-07-26 21:46:48.608 UTC [1177648] STATEMENT:  CREATE TRANSFORM FOR hstore LANGUAGE plperl (FROM SQL WITH FUNCTION internal_in(cstring), TO SQL WITH FUNCTION plperl_to_hstore(internal));
2023-07-26 21:46:48.609 UTC [1177648] ERROR:  transform for type hstore language "plperl" already exists
2023-07-26 21:46:48.609 UTC [1177648] STATEMENT:  CREATE TRANSFORM FOR hstore LANGUAGE plperl (FROM SQL WITH FUNCTION hstore_to_plperl(internal), TO SQL WITH FUNCTION plperl_to_hstore(internal));
2023-07-26 21:46:48.615 UTC [1177648] ERROR:  type "foo" does not exist
2023-07-26 21:46:48.615 UTC [1177648] STATEMENT:  DROP TRANSFORM FOR foo LANGUAGE plperl;
2023-07-26 21:46:48.615 UTC [1177648] ERROR:  language "foo" does not exist
2023-07-26 21:46:48.615 UTC [1177648] STATEMENT:  DROP TRANSFORM FOR hstore LANGUAGE foo;
2023-07-26 21:46:48.641 UTC [1177622] LOG:  checkpoint starting: immediate force wait
2023-07-26 21:46:48.672 UTC [1177622] LOG:  checkpoint complete: wrote 9 buffers (0.1%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.009 s, sync=0.006 s, total=0.032 s; sync files=7, longest=0.003 s, average=0.001 s; distance=4719 kB, estimate=4719 kB
2023-07-26 21:46:49.193 UTC [1177622] LOG:  checkpoint starting: immediate force wait
2023-07-26 21:46:49.308 UTC [1177622] LOG:  checkpoint complete: wrote 1808 buffers (11.0%); 0 WAL file(s) added, 0 removed, 1 recycled; write=0.038 s, sync=0.056 s, total=0.115 s; sync files=497, longest=0.021 s, average=0.001 s; distance=12486 kB, estimate=12486 kB
2023-07-26 21:46:49.331 UTC [1177622] LOG:  checkpoint starting: immediate force wait
2023-07-26 21:46:49.361 UTC [1177622] LOG:  checkpoint complete: wrote 2 buffers (0.0%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.005 s, sync=0.006 s, total=0.031 s; sync files=2, longest=0.005 s, average=0.003 s; distance=1 kB, estimate=11237 kB
2023-07-26 21:46:49.469 UTC [1177694] ERROR:  current user cannot be dropped
2023-07-26 21:46:49.469 UTC [1177694] STATEMENT:  DROP USER su;
2023-07-26 21:46:49.825 UTC [1177622] LOG:  checkpoint starting: immediate force wait
2023-07-26 21:46:49.907 UTC [1177622] LOG:  checkpoint complete: wrote 1036 buffers (6.3%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.033 s, sync=0.030 s, total=0.083 s; sync files=296, longest=0.008 s, average=0.001 s; distance=8732 kB, estimate=10987 kB
2023-07-26 21:46:49.942 UTC [1177622] LOG:  checkpoint starting: immediate force wait
2023-07-26 21:46:49.971 UTC [1177622] LOG:  checkpoint complete: wrote 8 buffers (0.0%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.006 s, sync=0.007 s, total=0.029 s; sync files=7, longest=0.006 s, average=0.001 s; distance=34 kB, estimate=9891 kB
2023-07-26 21:46:50.075 UTC [1177722] ERROR:  duplicate key value violates unique constraint "company_pkey"
2023-07-26 21:46:50.075 UTC [1177722] DETAIL:  Key (id)=(1) already exists.
2023-07-26 21:46:50.075 UTC [1177722] STATEMENT:  INSERT  INTO Company(ID, Name) VALUES (1, 'Percona');
2023-07-26 21:46:50.229 UTC [1177742] ERROR:  division by zero
2023-07-26 21:46:50.229 UTC [1177742] STATEMENT:  SELECT 1/0;
2023-07-26 21:46:50.230 UTC [1177742] ERROR:  relation "unknown" does not exist at character 15
2023-07-26 21:46:50.230 UTC [1177742] STATEMENT:  SELECT * FROM unknown;
2023-07-26 21:46:50.230 UTC [1177742] ERROR:  syntax error at or near "ELECET" at character 1
2023-07-26 21:46:50.230 UTC [1177742] STATEMENT:  ELECET * FROM unknown;
2023-07-26 21:46:50.231 UTC [1177742] WARNING:  warning message
2023-07-26 21:46:50.231 UTC [1177742] CONTEXT:  PL/pgSQL function inline_code_block line 3 at RAISE
2023-07-26 21:46:50.367 UTC [1177756] ERROR:  role "su" already exists
2023-07-26 21:46:50.367 UTC [1177756] STATEMENT:  CREATE USER su WITH SUPERUSER;
2023-07-26 21:46:50.389 UTC [1177756] ERROR:  permission denied for function pg_stat_monitor_reset
2023-07-26 21:46:50.389 UTC [1177756] STATEMENT:  SELECT pg_stat_monitor_reset();
